frontend/src/css/globals.css

```
@@ -1,12 +1,10 @@
 * {
     box-sizing: border-box;
-    user-select: none;
-
-    text-rendering: optimizeLegibility;
-
-    font-smoothing: always;
     -webkit-font-smoothing: subpixel-antialiased;
+    font-smoothing: always;
     -moz-osx-font-smoothing: grayscale;
+    user-select: none;
+    text-rendering: optimizeLegibility;
```

Stylelint (17.6.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)